.github/workflows/olympixStaticAnalysis.yml

```diff
@@ -0,0 +1,51 @@
+name: Olympix Static Analysis
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+    paths:
+      - 'src/**.sol'
+      - '.github/workflows/olympixStaticAnalysis.yml'
```

Comment:
why is the path of the git action itself here?

Reply:
removed